### PR TITLE
Adaption regarding new institute's CI

### PR DESCRIPTION
**Pull request by ckittl**
Work started by @bernharddick. Just made a few more adaptions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,38 @@
...
 To install this into your local TeXLive System so you can use it anywhere without copying file, use:
 ```
-$ git clone https://github.com/maxnoe/tudobeamertheme $(kpsewhich --var-value TEXMFHOME)/tex/latex/tudobeamertheme
+$ git clone https://github.com/ie3-institute/tudobeamertheme $(kpsewhich --var-value TEXMFHOME)/tex/latex/tudobeamertheme
+```
+
+Aforementioned command typically doesn't work on MikTeX systems.
+Therefore, locate the proper path, e.g. from MikTeX Console GuI and adapt the following line:
+```
+$ git clone https://github.com/ie3-institute/tudobeamertheme "c:/Program Files/MikTeX/tex/latex/tudobeamertheme"
```
Comment (johanneshiry) [inline README.md]:
```suggestion
$ git clone \
        https://github.com/ie3-institute/tudobeamertheme \
        "c:/Program Files/MikTeX/tex/latex/tudobeamertheme"
```

File: README.md [hunk 2]
```diff
@@ -1,15 +1,38 @@
...
+![](images/teaser.png)
 
 It follows the Corporate Design of the Powerpoint Templates the TU Dortmund supplies.
 
 Feel free to use it. License is BEERWARE.
 
 To install this into your local TeXLive System so you can use it anywhere without copying file, use:
 ```
-$ git clone https://github.com/maxnoe/tudobeamertheme $(kpsewhich --var-value TEXMFHOME)/tex/latex/tudobeamertheme
+$ git clone https://github.com/ie3-institute/tudobeamertheme $(kpsewhich --var-value TEXMFHOME)/tex/latex/tudobeamertheme
```
Comment (johanneshiry) [inline README.md]:
```suggestion
$ git clone \
            https://github.com/ie3-institute/tudobeamertheme \
            $(kpsewhich --var-value TEXMFHOME)/tex/latex/tudobeamertheme
```

**Review by johanneshiry — DISMISSED** [review]
Thanks for your work. LGTM.

Approved, but added two comments to avoid sliders in README.md due to too long lines. Feel free to accept or decline.

**Comment by ckittl** [comment]
> Thanks for your work. LGTM.
> 
> 
> 
> Approved, but added two comments to avoid sliders in README.md due to too long lines. Feel free to accept or decline.

Looks good! Took your adaptions!

**Review by johanneshiry — APPROVED** [review]

**Pull request merged**